src/messagesView/MessagesTable.js

```diff
@@ -1,9 +1,5 @@
 import TelemetryTable from 'openmct.tables.TelemetryTable'
 export default class MessagesTable extends TelemetryTable {
-    constructor(domainObject, openmct){
```

Comment:
do we need to pass in table options for this?

Reply:
good catch. i've modified the view provider. this constructor remains deleted.